src/ios/CDVWKInAppBrowser.m

```diff
@@ -785,7 +787,7 @@ - (void)didFinishNavigation:(WKWebView*)theWebView
 
         [self.cordovaPluginResultProxy sendOKWithMessageAsDictionary:@{@"type":@"loadstop", @"url":url}];
         if ([windowState isUnhiding]) {
-            [self showWithHiddenAs:NO withNoAnimate:true];
+//            [self showWithHiddenAs:NO withNoAnimate:YES];
```

Comment:
This needs uncommenting. We never attempted to navigate after opening so this was never ran.